wordstats/language_data/t2l/code/getDictonary.sh

```diff
@@ -0,0 +1,65 @@
+# Explanations need to be added and the code as a whole still needs to be refactored
```

Comment:
either this comment is wrong, and then remove it
or it is right, and then please act on it! 
thanks!